orc8r/cloud/helm/orc8r/charts/nms/templates/helpers/labels.tpl

```diff
@@ -33,3 +33,8 @@ app.kubernetes.io/name: {{ $application }}
 app.kubernetes.io/component: {{ $component }}
 app.kubernetes.io/instance: {{ $envAll.Release.Name }}
 {{- end -}}
+
+{{/* Generate selector labels */}}
+{{- define "magmalte-image-version-label" -}}
```

Comment:
Should FEG components of the Orc8r be considered here, either as part of this PR or a new one?

Reply:
Right, we'll want to apply these changes to all the Orc8r modules, either here or subsequent PR https://magma.github.io/magma/docs/next/orc8r/dev_dependencies